logging/src/main/java/org/jboss/as/logging/logmanager/ConfigurationPersistence.java

```diff
@@ -377,6 +377,9 @@ public void writeConfiguration(final OperationContext context) {
                 loggingConfig = FileResolver.resolvePath(context, "jboss.server.data.dir", PROPERTIES_FILE);
                 break;
             }
+            case EMBEDDED_SERVER: {
+                return;
+            }
```

Comment:
I think this should be added, I don't think it solves the issue though. Testing locally I still see log output. It's possible the `$JBOSS_HOME/domain/configuration/logging.properties` is being found on the class path which is why it's displaying log messages. Logging does seem relevant, but I can see where we don't want it output to the console. I'd have to look further into what `--std-out=discard` actually does.